releases/evm/2025-01-ITS-v2.1.0.md

```diff
@@ -47,7 +47,7 @@ Amplifier ITS
 
 ```bash
 # Deploy new implementation
-node evm/deploy-its.js -s "v2.1.0 devnet-amplifier" -m create2 --reuseProxy
+node evm/deploy-its.js -s "ITS v2.1.0 devnet-amplifier" -m create2 --reuseProxy
```

Comment:
`ITS` string is prefixed somewhere in code before outputted in json file. Prefixing it here will result in  something like `ITS ITS v2.1.0 devnet-amplifier`